---
string position  # The current rollerbrush position (up/down)
